processing/src/main/java/org/apache/druid/segment/join/HashJoinSegmentStorageAdapter.java

```diff
@@ -237,22 +218,16 @@ public Metadata getMetadata()
       @Nullable final QueryMetrics<?> queryMetrics
   )
   {
-
     final List<VirtualColumn> preJoinVirtualColumns = new ArrayList<>();
     final List<VirtualColumn> postJoinVirtualColumns = new ArrayList<>();
+
     final Set<String> baseColumns = determineBaseColumnsWithPreAndPostJoinVirtualColumns(
```

Comment:
With the deletion below, CI (spotbugs and intellij inspections) is flagging this as unused now